telegram/_inline/inlinequery.py

```diff
@@ -123,13 +129,14 @@ async def answer(
         next_offset: str = None,
         switch_pm_text: str = None,
         switch_pm_parameter: str = None,
+        current_offset: str = None,
+        *,
+        auto_pagination: bool = False,
```

Comment:
can you also change the Args: to Keyword Args in the docs for this method (InlineQuery.answer())?